templates/project/.github/workflows/frontend.yaml.twig

```diff
@@ -38,3 +38,6 @@ jobs:
 
             - name: Run Webpack Encore
               run: yarn encore production
+
+            - name: Check if the compilation is up to date
+              run: git diff --no-patch --exit-code src/Resources/public/app.js src/Resources/public/app.css
```

Comment:
We don't need this for every project

Reply:
Is there a way to check for everything in web/Resources/public? Every project with frontend enabled should check for that directory (atm only admin bundle)

Reply:
@core23 We need this for every project with the frontend github actions.

@jordisala1991 I made the change